lib/screens/v2/alerts_widget.ex

```diff
@@ -0,0 +1,19 @@
+defprotocol Screens.V2.AlertsWidget do
+  @moduledoc """
+  Protocol for a widget that models zero or more alerts.
```

Comment:
In the moduledoc for these two different AlertsWidgets, could you add the widgets that qualify so far? Like, "for example: ". (We don't need to keep that list updated over time.)